weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -10,11 +10,16 @@
 from weave.trace.env import weave_trace_server_url
 from weave.trace_server import requests
 from weave.trace_server import trace_server_interface as tsi
-from weave.trace_server.async_batch_processor import AsyncBatchProcessor
+from weave.trace_server_bindings.async_batch_processor import AsyncBatchProcessor
 from weave.wandb_interface import project_creator
 
 logger = logging.getLogger(__name__)
 
+# Default timeout values (in seconds)
+DEFAULT_CONNECT_TIMEOUT = 10
+DEFAULT_READ_TIMEOUT = 30
```

Comment:
seems low actually - how did you choose 30?

Reply:
It seemed reasonable -- are you concerned about large queries timing out?